.github/workflows/test-and-publish.yaml

```
@@ -135,7 +135,7 @@ jobs:
         with:
           path: |
             ./_spidermonkey_install/*
-          key: spidermonkey-${{ runner.os }}-${{ runner.arch }}
+          key: spidermonkey102.13-${{ runner.os }}-${{ runner.arch }}
```

Not a release blocker:
We should change our build system to be able to specify a SpiderMonkey version from the CI 
(possibly via environment variable `SPIDERMONKEY_VERSION`)

```suggestion
          key: spidermonkey${{ env.SPIDERMONKEY_VERSION }}-${{ runner.os }}-${{ runner.arch }}
```